src/DotNest.Core.SDK.Web/Recipes/DotNest.Core.SDK.Setup.recipe.json

```diff
@@ -0,0 +1,72 @@
+﻿{
+  "name": "DotNest.Core.SDK.Setup",
+  "displayName": "DotNest Core SDK - Setup",
+  "description": "",
+  "author": "DotNestCoreSDK",
+  "website": "https://www.dotnestcoresdk.com",
+  "version": "1.0",
+  "categories": [ "dotnestcoresdk" ],
+  "tags": [],
+
+  "steps": [
+    {
+      "name": "feature",
+      "disable": [],
+      "enable": [
+        // Ochard Core
+        "OrchardCore.Admin",
```

Comment:
This isn't the full list of features that can be turned on. Please check [this file.](https://github.com/Lombiq/DotNest-Tenants-Core/blob/dev/src/Modules/DotNest.Hosting.Tenants/Recipes/AllModules.recipe.json)